dev/AppLifecycle/ActivationRegistrationManager.cpp

```diff
@@ -11,14 +11,9 @@
 #include "ExtensionContract.h"
 #include "EncodedLaunchExecuteCommand.h"
 
-namespace winrt::Microsoft::ApplicationModel::Activation::implementation
+namespace winrt::Microsoft::Windows::AppLifecycle::implementation
 {
-    std::wstring GenerateEncodedLaunchUri(std::wstring const& appUserModelId, std::wstring const& contractId)
-    {
-        Windows::Foundation::Uri uri{ wil::str_printf<std::wstring>(L"%s://%s?%s=%s",
-            c_encodedLaunchSchemeName, appUserModelId.c_str(), c_contractIdKeyName, contractId.c_str()).c_str() };
-        return uri.AbsoluteUri().c_str();
-    }
+    using namespace winrt::Windows::Foundation;
 
     std::wstring GenerateCommandLine(std::wstring const& modulePath)
```

Comment:
This code does two levels of substitution with printf specifiers and is really hard to reason about.  Can you put a short example comment of what this does?